### ISSUE TITLE
Missing crs on hvmv_substation_voronoi table in CI

### ISSUE THREAD

**Issue by fwitte** [openego/eGon-data#515]
An error message is thrown when reading the hvmv_substation_voronoi table:

```
Traceback (most recent call last):
  File "/home/egondata/git-repos/friday-evening-weekend-run/environment/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/egondata/git-repos/friday-evening-weekend-run/code/src/egon/data/datasets/__init__.py", line 195, in skip_task
    result = super(type(task), task).execute(*xs, **ks)
  File "/home/egondata/git-repos/friday-evening-weekend-run/environment/lib/python3.8/site-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/home/egondata/git-repos/friday-evening-weekend-run/environment/lib/python3.8/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/egondata/git-repos/friday-evening-weekend-run/code/src/egon/data/datasets/hydrogen_etrago/storage.py", line 267, in calculate_and_map_saltcavern_storage_potential
    substation_voronoi = db.select_geodataframe(
  File "/home/egondata/git-repos/friday-evening-weekend-run/code/src/egon/data/db.py", line 205, in select_geodataframe
    gdf = gpd.read_postgis(
  File "/home/egondata/git-repos/friday-evening-weekend-run/environment/lib/python3.8/site-packages/geopandas/geodataframe.py", line 1275, in to_crs
    geom = df.geometry.to_crs(crs=crs, epsg=epsg)
  File "/home/egondata/git-repos/friday-evening-weekend-run/environment/lib/python3.8/site-packages/geopandas/geoseries.py", line 1120, in to_crs
    self.values.to_crs(crs=crs, epsg=epsg), index=self.index, name=self.name
  File "/home/egondata/git-repos/friday-evening-weekend-run/environment/lib/python3.8/site-packages/geopandas/array.py", line 763, in to_crs
    raise ValueError(
ValueError: Cannot transform naive geometries.  Please set a crs on the object first.
```

The `db.select_geodataframe` method complains about missing crs. QGIS, however, shows that the table 
has a crs specified (`epsg=4326`). Any idea, where this error message comes from? It is caused by the task `hydrogen_etrago.storage.calculate-and-map-saltcavern-storage-potential` when reading the substation voronoi data. This did not happen on my machine and it did not happen on last week's run (the failing task was due to the geopandas version issue #504, but after manually fixing that, the task ran smoothly).

**Comment by fwitte** [comment]
After clearing the task, it ran successful... There might be some dependency issue..?

**Comment by fwitte** [comment]
It was indeed a missing dependency, will be fixed by #478
